### PR TITLE
Update changelog to include breaking change trace metric tag sampler_type

### PR DESCRIPTION
**Pull request by guanw**
Signed-off-by: Jude Wang <judew@uber.com>

## Which problem is this PR solving?
- Updates https://github.com/jaegertracing/jaeger/issues/1592

## Short description of the changes
- Update breaking changes for new `sampler_type` tag for traces metrics

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ Changes by Version
 
 #### Backend Changes
 
+- The traces related metrics on collector now have a new tag `sampler_type` ([#1576](https://github.com/jaegertracing/jaeger/pull/1576))
+
+  this might break some existing metrics dashboard (if so, users need to update query to aggregate over this new tag).
+
```
Comment (yurishkuro) [inline CHANGELOG.md]: should this be under Breaking changes then?  

cc @pavolloffay @objectiser

Comment (guanw) [reply CHANGELOG.md]: hmmm. Like what @black-adder mentioned in that PR, it would depends on how user configured their dashboard at first place.

Comment (guanw) [reply CHANGELOG.md]: Updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@ Changes by Version
 
 ##### Breaking Changes
 
+- The traces related metrics on collector now have a new tag `sampler_type` ([#1576](https://github.com/jaegertracing/jaeger/pull/1576))
```
Comment (yurishkuro) [inline CHANGELOG.md]: Can we list all the affected metrics? Eg grep prom endpoint for sampler_type

Comment (pavolloffay) [reply CHANGELOG.md]: @guanw ping? We would like to release this week.

Comment (guanw) [reply CHANGELOG.md]: sry was working on something else and got carried. away. Will update shortly

**Pull request merged**